frontend/src/Components/Account/Login.js

```diff
@@ -1,5 +1,103 @@
-import React from "react";
+import React, { useState } from "react";
+import { navigate, A } from "hookrouter";
+import axios from "axios";
 
 export default function Login() {
-  return <div>Login Page</div>;
+  const initForm = {
+    email: "",
+    password: "",
+  };
+  const [form, setForm] = useState(initForm);
+  const [formError, setFormError] = useState(false);
+
+  const handleChange = (e) => {
+    const { value, name } = e.target;
+    const FieldValue = { ...form };
+    FieldValue[name] = value;
+    setForm(FieldValue);
+  };
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    axios
+      .post("http://localhost:8000/user/login", { ...form })
+      .then((resp) => {
+        localStorage.setItem("access_token", resp.data.data.access_token);
+      })
+      .catch((err) => {
+        setFormError(true);
+      });
+  };
+
+  return (
+    <div>
+      <div className="flex items-center justify-center py-5 px-4 sm:px-6 lg:px-8">
+        <div className="max-w-md w-full">
+          <form
+            onSubmit={handleSubmit}
+            className="bg-white shadow rounded px-8 pt-6 pb-8 my-5 lg:my-20"
+          >
+            <div className="mb-4">
+              <label
+                className="block text-gray-700 text-sm font-bold mb-2"
+                htmlFor="email"
+              >
+                Email
+              </label>
+              <input
+                aria-label="user name"
+                name="email"
+                value={form.email}
+                type="email"
+                onChange={handleChange}
+                className="appearance-none border rounded w-full py-2 px-3 text-gray-700 bg-gray-200 leading-tight focus:outline-none focus:shadow-outline"
+                placeholder="Email"
+              />
+            </div>
+            <div className="mb-2">
+              <label
+                className="block text-gray-700 text-sm font-bold mb-2"
+                htmlFor="password"
+              >
+                Password
+              </label>
+              <input
+                aria-label="Password"
+                name="password"
+                type="password"
+                value={form.password}
+                onChange={handleChange}
+                className="appearance-none border rounded w-full py-2 px-3 text-gray-700 bg-gray-200 leading-tight focus:outline-none focus:shadow-outline"
+                placeholder="******************"
+              />
+            </div>
+
+            {formError && (
+              <div className="text-red-400 font-semibold">
+                Invalid Credentials
```

Comment:
Will be solved by https://github.com/Open-Sourced-Olaf/SocioMark/issues/32, 👍🏼 for now